tasks/fh-build.js

```diff
@@ -343,31 +374,35 @@ module.exports = function(grunt) {
     grunt.task.run('env:fh');
 
     if (this.target === 'dist') {
-      grunt.task.run(['clean:fh-dist', 'fh:make-version-file', 'copy:fh-dist']);
+      grunt.task.run(['clean:fh-dist', 'fh:make-version-file', 'shell:ensureDist', 'shell:pack']);
 
       if (checkPlaceholderFileExists()) {
         grunt.task.run('fh-generate-dockerised-config');
       }
 
       if (!grunt.option('only-bundle-deps')) {
-        grunt.task.run('compress:fh-dist');
+        // grunt.task.run('compress:fh-dist');
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ACommentedCode)